src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return (century)
+    ? (people
+      .filter((x) => (Math.ceil(x.died / 100) === century && x.sex === 'm'))
```

Comment:
Let's move filtered people to a variable. Than you can use it everywhere you need to. DRY - don't repeat yourself.